internal/config/config.go

```diff
@@ -139,6 +139,8 @@ type TypeConfig struct {
 	// GenerateStructGetters enables the auto-generation of field getters for all fields on a struct.
 	// i.e. if a struct has a field `name` then a function would be created called `GetName()`
 	GenerateStructGetters bool `yaml:"generate_struct_getters,omitempty"`
+
+	StructTags []string `yaml:"struct_tags,omitempty"`
```

Comment:
@zlesnr remind me, did we want to add `FieldStructTags []string` as well to this?

Reply:
Hypothetical example:

```diff
+  // Applies to all fields of the struct. Use FieldTagsOverride to change a specific field's tags.
   StructTags []string `yaml:"struct_tags,omitempty"`

+  // FieldTagOverrides does something but pondering the best way to implement
+  FieldTagOverrides []FieldTagOverride `yaml:"field_tag_overrides,omitempty"`
```

This can also be done as an enhancement in another PR.

Reply:
If we have a use case right now, but if not maybe we hold off.  Up to you though.

Reply:
I think we merge and update if/when we run into the issue since it's not a deal breaker by any means.